util/dcrtime_test.go

```diff
@@ -38,6 +39,14 @@ func TestTimestamp(t *testing.T) {
 	filename := tempFile(512)
 	defer os.Remove(filename)
 
+	// sets mime types map for validation
```

Comment:
This should be the default list that is set somewhere. I keep seeing this copy and paste and it is not ok.

Reply:
So basically we have this list set up here in this test file, and another on politeiad/config.go, both lists being equal, containing the three valid default mime types. Where could we set this so that both files can import that list? I tried to import `defaultMimeTypes` from config.go, but Im not sure if go packages allows us to export vars.

Reply:
Hey @marcopeereboom, now I'm setting the default on our `api/v1/mime` package and importing it where its needed, basically in the config and dcrtime_test.